### PR TITLE
fixed external tab issue

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Rajdave69 — COMMENTED** [review]
What was the need to use two lines (13 and 14) when the same could be done in one? Looks good otherwise.

**Comment by ghost** [comment]
> What was the need to use two lines (13 and 14) when the same could be done in one? Looks good otherwise.

Thanks for figuring it out.

File: code/resources/scripts/components/server/PluginContainer.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import TitledGreyBox from '@/components/elements/TitledGreyBox';
 import FlashMessageRender from '@/components/FlashMessageRender';
 import getPlugins, { Plugin } from '@/api/server/plugins/getPlugins';
 import ServerContentBlock from '@/components/elements/ServerContentBlock';
-import installPlugin from '@/api/server/plugins/installPlugin';
```
Comment (camwhi-te) [inline code/resources/scripts/components/server/PluginContainer.tsx:13]: This import needs to be present, it is used within this file.

File: code/resources/scripts/components/server/PluginContainer.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ export default () => {
                                                         <Button className={'m-1'} onClick={() => setOpen(true)}>
                                                             <FontAwesomeIcon icon={faCloudDownloadAlt} fixedWidth />
                                                         </Button>
-                                                        <a href={`https://api.spiget.org/v2/resources/${plugin.id}/go`}>
+                                                        <a href={`https://api.spiget.org/v2/resources/${plugin.id}/go`} target="_blank">
```
Comment (camwhi-te) [inline code/resources/scripts/components/server/PluginContainer.tsx]: Please format this as `target={'_blank'}`

**Review by camwhi-te — CHANGES_REQUESTED** [review]
Please review comments

**Comment by ghost** [comment]
> Please review comments

Done

**Pull request closed without merging**